### DISCUSSION TITLE
Set cluster_id in Clustermetadata

### DISCUSSION THREAD

**Discussion by astream2018** [strimzi/strimzi-kafka-operator#6348] [Q&A]
I want to startup a Strimzi cluster and I would like to set my own Cluster_Id (ClusterMetadata).
Is this even possible using when Strimzi?

**Comment by scholzj** [comment]
No, this is not supported. I'm also not aware of any official Kafka-way to do this to possibly add support for it (although I'm not sure what the value of it would be anyway).
